### PR TITLE
Add ability to run specific plugin

### LINKED ISSUE #18: [FEATURE] Allow comeback to a specific plugin
**Describe the solution you'd like**
The user should be able to use a specific plugin from the `.comeback` file.

`comeback ipython` Should run only the ipython plugin.

### PR DESCRIPTION
**Pull request by agamm**
Added the ability to run a specific plugin instead from the .comeback recipe.
Example: `comeback -v run vscode cwd=.`
Will be verbose, run the `vscode` plugin with the parameters: `cwd=.` 

Also update pipenv and pip.

fixes https://github.com/agamm/comeback/issues/18

### CODE REVIEW

File: comeback/main.py [hunk 1]
```diff
@@ -99,17 +111,31 @@ def main():
...
         return
 
     main()
+
+
+@cli.command()
+@click.argument('plugin', required=True)
+@click.argument('plugin_params', required=False)
+def run(plugin, plugin_params):
+    print(plugin, plugin_params)
```
Comment (yammesicka) [inline comeback/main.py]: Remove the `print`?

File: comeback/main.py [hunk 2]
```diff
@@ -99,17 +111,31 @@ def main():
...
+    verbose_echo("Running specific plugin.")
```
Comment (yammesicka) [inline comeback/main.py]: Maybe should be like?:
> f'Running plugins: {plugin} with args: {args}'

Also, `'`

File: comeback/main.py [hunk 3]
```diff
@@ -22,6 +22,18 @@ def get_probable_project_name():
     return get_cwd().name
 
 
+def parse_args(args_string):
+    if not args_string:
+        return {}
+
+    ret_args = {}
+    for arg in args_string.split(","):
+        key_value = arg.split("=")
```
Comment (yammesicka) [inline comeback/main.py:31]: We should use `.partition()` instead, to prevent problems like `arg=(compiler=vs)` or some other weird corner cases.

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: comeback/main.py (hunk 1 above)
Comment (agamm) [reply comeback/main.py]: +1

File: comeback/main.py (hunk 2 above)
Comment (agamm) [reply comeback/main.py]: Is there any way to tell pycharm to make them always single quote instead of double?

Comment (yammesicka) [reply comeback/main.py]: I don't know, vim ftw

File: comeback/main.py (hunk 3 above)
Comment (agamm) [reply comeback/main.py:31]: Didn't really get the use-case. Open an issue and rewrite this?

Comment (yammesicka) [reply comeback/main.py:31]: 👍

**Review by yammesicka — APPROVED** [review]

**Pull request merged**